scribble-doc/scribblings/scribble/config.scrbl

```diff
@@ -640,6 +640,25 @@ style:
 
  ]
 
+Additionally, the @filepath{racket.tex} Latex configuration
+includes several macros that you can redefine to adjust the
+output style of Racket code:
+
+@itemlist[
+ @item{@ltxd[2]{SColorize} --- Sets the color scheme of
+  Racket code. Can be redefined to create black and white
+  code. The first argument is the requested color, and the
+  second argument is the text for that color.}
+ @item{@ltxd[1]{SHyphen} --- Enables or Disables the ability
+  for identifiers and keywords in Racket code from being
+  hyphenated. Defaults to enabled (for compatibility).
+  Redefine to disable or change the hyphenation behavior. For
+  example, to cause the text to overfill rather than hyphen,
+  it can be redefined to:
+  @tt["\\renewcommand{\\SHyphen}[1]{\\mbox{#1}}"]. The first
+  argument is an identifier or keyword inside of a code
+  block.}]
```

Comment:
Ah, okay. No, I'd rather leave this as is because while it makes sense to enable/disable hyphens, there's lots of different strategies that may or may not be okay for your document. You might be okay with, say, text box overflows, or you might prefer underflows, or you might want an abbreviation (although I admit I'm stretching on that one.)